docs/README.md

```diff
@@ -65,3 +73,6 @@ Repo name in the format of `<owner>/<repoName>`. Defaults to the repo that execu
 | false    | empty string |
 
 Specifies a pattern to match. If not specified then every release will be targeted. If specified then every release containing the pattern will be targeted. Use this option for example to remove old beta releases.
```

Comment:
Need to add the Regex term to make this accurate and clear now.

"Specifies a Regex pattern..."